src/CSSBattle/1_SimplySquare/kiyong.html

```diff
@@ -0,0 +1,13 @@
+<div class="box"></div>
```

Comment:
Didn't load in website. Might be named wrongly

Reply:
resolved in commit a9937a9